doc/man/man1/kdb-elektrify-getenv.1

```diff
@@ -1,7 +1,7 @@
 .\" generated with Ronn/v0.7.3
 .\" http://github.com/rtomayko/ronn/tree/0.7.3
 .
-.TH "KDB\-ELEKTRIFY\-GETENV" "1" "February 2018" "" ""
```

Comment:
Please do not add those pointless changes. If you want to get rid of these files, below is a Fish script I use to do just that:

```fish
for file in (git ls-files -m)
	if echo "$file" | egrep -q '\.[17]$' && 
	   test (git diff --numstat "$file" | cut -f1) -le 1 && 
	   git diff "$file" | grep -q '^-.TH';
		git checkout "$file"
	end
end
```

.

Reply:
Thank you for pointing that out. The script, however, did not work for me:

```
Expected end of the statement, but instead found a '&'
scripts/git-checkout-ronn-files (line 4):       if echo "$file" | egrep -q '\.[17]$' && 
                                                                                      ^
<W> fish: Error while reading file scripts/git-checkout-ronn-files
```

fish, version 2.4.0

Reply:
Yeah, it requires the latest unreleased version of Fish (3.0) 😊. Below is a `bash` version of the same script:

```bash
#!/usr/bin/env bash

for file in $(git ls-files -m); do
	if echo "$file" | egrep -q '\.[17]$' && \
	   test $(git diff --numstat "$file" | cut -f1) -le 1 && \
	   git diff "$file" | grep -q '^-.TH'; then
		git checkout "$file"
	fi
done
```

.